docs/resources/workspace_current_state.md

```diff
@@ -0,0 +1,28 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "tharsis_workspace_current_state Resource - terraform-provider-tharsis"
+subcategory: ""
+description: |-
+  Defines and manages a workspace current state.
+---
+
+# tharsis_workspace_current_state (Resource)
```

Comment:
From a user's perspective, I would think `tharsis_workspace_apply` would be the appropriate resource name. If `apply` is odd, I'm up for suggestions but `current_state` suggests a data source to get information.

Reply:
Agreed that "current_state" is not the best name for it.  How about tharsis_workspace_application or tharsis_workspace_applied to keep the last part a noun rather than a verb?

Reply:
We call them runs in Tharsis, so we should probably go with `run`. In this case `run` is a noun.

Reply:
Since this isn't handling the lifecycle of a run or apply I'd like to use a different name since it's technically just creating an apply to get to the desired state. This new resource is really indicating that the workspace should have a current state that was deployed using the specified module and version. `tharsis_workspace_current_state` was just a placeholder until we come up with a better name. We could call it `tharsis_workspace_deployed_module`.